lib/rust/ensogl/core/src/timer/timeout.rs

```diff
@@ -0,0 +1,84 @@
+//! FRP bindings for browser `setTimeout` timer.
+
+use crate::prelude::*;
+
+use enso_web::traits::ClosureOps;
+use enso_web::window;
+use enso_web::Closure;
+
+use std::time::Duration;
+
+/// Closure type alias for use in `setTimeout` call.
+type TimerClosure = Closure<dyn FnMut()>;
+
+/// A one-shot timer based on `setIntevral` browser API.
```

Comment:
There are typos everywhere in this PR. Please be sure to fix all typos and grammar errors before making PR next time.